cliquet/utils.py

```diff
@@ -147,7 +147,21 @@ def hmac_digest(secret, message, encoding='utf-8'):
 
 def dict_subset(d, keys):
     """Return a dict with the specified keys"""
-    return {k: d[k] for k in keys if k in d}
+    result = {}
+
+    for key in keys:
+        if '.' in key:
```

Comment:
Since python 2.6, the stdlib [`operator.attrgetter`](https://docs.python.org/2/library/operator.html#operator.attrgetter) has been added the capability of parsing dotted attributes, we might want to use this here.